script/fetch-issues.js

```diff
@@ -2,6 +2,7 @@ const fs = require('fs');
 const moment = require('moment');
 const GraphQL = require('graphql-client');
 const sleep = require('sleep');
+const { DEPLOY_PREVIEW } = require('netlify-env');
```

Comment:
A little weird, but instead of setting `isDeployPreview = DEPLOY_PREVIEW` below, you could just do:

```javascript
const { DEPLOY_PREVIEW: isDeployPreview } = require('netlify-env');
```

Up to you, though.